qiita_pet/handlers/api_proxy/artifact.py

```diff
@@ -73,8 +76,9 @@ def artifact_post_req(user_id, filepaths, artifact_type, name,
     ----------
     user_id : str
         User adding the atrifact
-    filepaths : dict of {str: [str, ...], ...}
-        List of files to attach to the artifact, keyed to the file type
+    filepaths : str
```

Comment:
This description doesn't match "my inferred type", it seems like it really is a dictionary, not a string, am I missing something?